app/decorators/user_decorator.rb

```diff
@@ -88,9 +72,32 @@ def assigned_color
         text: "#c9d2dd"
       },
     ]
+    colors |= white_text
     colors[id % 10]
   end
 
+  def white_text
+    white = [
```

Comment:
At this point you don't need the variable, the method can just return the array.
Which made me think you might even want to move it as a constant and freeze it:

```ruby
class UserDecorator
  WHITE_TEXT_COLORS = [
  # ...
  ].freeze
end
```

and then use it in `assigned_color`